src/Modules/OrchardCore.Commerce/Abstractions/IRegionService.cs

```diff
@@ -0,0 +1,17 @@
+using System.Collections.Generic;
+using System.Globalization;
+using System.Threading.Tasks;
+
+namespace OrchardCore.Commerce.Abstractions;
+
+/// <summary>
+/// When implemented a service for handling regions.
```

Comment:
"When implemented" is redundant for an interface. It's terminology usually used in abstract types to indicate that implementation depends on the child. I know I have used it in `IPaymentProvider` too, that was also wrong of me.

```suggestion
/// A service for accessing and customizing regions.
```